src/containers/search/detailList.js

```diff
@@ -19,7 +20,9 @@ class DetailList extends Component {
       let valueNode;
       let value = d[field];
 
-      if (Array.isArray(value)) {
+      if (field === 'crossReferenceLinks' && value) {
+        valueNode = ( value.map(ref => <DataSourceLink key={ref.displayName + ref.name + ref.url} reference={ref} />).reduce((a,b) => [a,', ',b]));
```

Comment:
I think you could use the `CommaSeparatedList` component here instead of the funky `reduce` thing